src/components/Banner/Banner.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { Button } from '@carbon/react';
+import { ArrowRight } from '@carbon/react/icons';
+
+import {
+  banner,
+  bannerText,
+  bannerHeading,
+  bannerDetails,
+  buttonBanner,
+} from './Banner.module.scss';
+
+const Banner = () => {
+  return (
+    <div aria-label="banner" className={banner}>
+      <p className={bannerText}>
+        <strong className={bannerHeading}>
+          A new web experience is coming!
+        </strong>
+        <span className={bannerDetails}>
+          The v1 Carbon platform v1 is now available in public beta.
```

Comment:
Duplicate "v1", and should platform be capitalized too?

```suggestion
          The Carbon Platform v1 is now available in public beta.
```

Reply:
Good catch, @aagonzales can you confirm correct copy here, the figma file has several versions depending on the breakpoint. I think it should probably be `The v1 Carbon platform is now available in public beta.` but wanted to double check. 
![Screen Shot 2022-10-05 at 8 56 22 AM](https://user-images.githubusercontent.com/2753488/194078458-bd2b5d25-5770-4297-b0a5-fb7f1f06761c.png)

Reply:
oh sorry for the confusion, I should have updated the image in the issue. This is the final copy with suggestions from Aubrey. 

> A new web experience is coming! The v1 Carbon platform is now available in public beta.
> CTA: Try the Carbon Platform